_patternlab-files/index.twig

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html id="patternlab-html">
+<head id="patternlab-head">
+	<title id="title">Pattern Lab</title>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width" />
+	<link rel="stylesheet" href="styleguide/css/styleguide.css" media="all" />
+	<link rel="stylesheet" href="styleguide/css/styleguide-specific.css" media="all" />
+	<link rel="stylesheet" href="styleguide/css/vendor/typeahead.css" media="all" />
+	<link rel="stylesheet" href="styleguide/css/vendor/prism.css" media="all" />
+	<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.0.3/jquery.min.js"></script>
+	<script>window.jQuery || document.write('<script src="styleguide/js/vendor/jquery.js"><\/script>')</script>
+</head>
+<body  id="patternlab-body">>
+	<!--Style Guide Header-->
+	<header class="sg-header" role="banner">
+		<a href="#sg-nav-container" class="sg-nav-toggle">Menu</a>
+		<div class="sg-nav-container" id="sg-nav-container">
+			{{ patternNav }}
+			{{ ishControls }}
+		</div>
+	</header>
+	<!--End Style Guide Header-->
+	
+	<!-- Iframe -->
+	<div id="sg-vp-wrap">
+		<div id="sg-cover"></div>
+		<div id="sg-gen-container">
+			<iframe id="sg-viewport" sandbox="allow-same-origin allow-scripts"></iframe>
+			<div id="sg-rightpull-container">
+				<div id="sg-rightpull"></div>
+			</div>
+		</div>
+	</div>
+	<!--end iFrame-->
+
+	<!-- the template for the code view slider -->
+	<script type="text/html" id="code-template">
```

Comment:
@sergesemashko if we remove the need for the sizing-controls, how much easier can this become? Technically, we don't need these for this iteration, but if they work without having to do too much, we could just leave them.